_data/trans/ru.json

```diff
@@ -26,16 +26,16 @@
     "morecontrib": "Другие участники проекта",
     "name": "Pусский",
     "noinfo": "Информация недоступна",
-    "popular": "Popular",
+    "popular": "Популярные",
     "sendmail": "E-mail отправлен",
     "showinfo": "показать информацию...",
-    "submit": "Submit a site",
+    "submit": "Отправить сайт",
     "tagline": "Каталог прямых ссылок для удаления аккаунта с веб-сайтов",
     "title": "Just Delete Me | Каталог прямых ссылок для удаления вашего аккаунта с различных веб-сайтов",
     "translationcontrib": "Участники перевода",
     "viewcontrib": "Просмотреть список всех участников проекта",
     "whatisthis": "Что это?",
-    "whatisthis1": "Многие компании используют технологию <a href=\"http://darkpatterns.org/\">dark pattern</a> для того, чтобы усложнить процесс удаления аккаунта. Каталог ссылок JustDeleteMe призван помочь вам с легкостью удалить аккаунт с веб-сайта",
-    "whatisthis2": "Есть сайт, который неободимо добавить? <a href=\"https://github.com/jdm-contrib/jdm\">Fork the project on GitHub</a>.",
+    "whatisthis1": "Многие компании используют технологию <a href=\"//darkpatterns.org/\">dark pattern</a> для того, чтобы усложнить процесс удаления аккаунта. Каталог ссылок JustDeleteMe призван помочь вам с легкостью удалить аккаунт с веб-сайта",
```

Comment:
If we're going to change the schema here, it should be changed to have an explicit `https://` rather than using a protocol-relative URL.

Reply:
Done. How all links are on `https://`